src/styles/Components/src/tokens-divider.css

```diff
@@ -0,0 +1,8 @@
+:root {
+  --denhaag-divider-border-color: var(--denhaag-grey-2);
+  --denhaag-divider-border-width: 1px;
+  --denhaag-divider-margin-block-start: 16px;
+  --denhaag-divider-margin-block-end: 16px;
+  --denhaag-divider-margin-inline-start: 16px;
+  --denhaag-divider-margin-inline-end: 16px;
```

Comment:
Perhaps it would be nice to add aggregation tokens for `-block` and `-inline`, since by default they're all 16px. We would still have the `-start` and `-end` specific tokens, so users _can_ overwrite them if they want, but have them default to a `denhaag-divider-margin-{block | inline}: 16px` token.

The current tokens would look like this:

```css
--denhaag-divider-margin-block-start: var(--denhaag-divider-margin-block); 
--denhaag-divider-margin-block-end: var(--denhaag-divider-margin-block); 
--denhaag-divider-margin-inline-start: var(--denhaag-divider-margin-inline); 
--denhaag-divider-margin-inline-end: var(--denhaag-divider-margin-inline);
```

@Robbert @mjcarsjens please correct me if it's a rubbish idea :P

Reply:
I mean, personally I do not think defining aggregation token for default margins adds any value here. Also, in your example, if someone decides they want `-inline-start` to be 32px, they may get confused seeing yet another design token.

Reply:
As a developer, I'd rather define just two tokens (the aggregated tokens) than 4 tokens (start, end for both block and inline). Especially if they're all the same, which is fairly likely.

Reply:
I am unsure whether or not this adds value. I mean, a dev could just as easily define his own aggregated tokens and insert those, that makes it easier for them to maintain rather than digging through our tokens. In my opinion this is a sort of "less is more" scenario. The aggregated tokens may help in more clearly defining things in our library, but leaving it this way I think leaves more room for customization. Curious what @Robbert and @mjcarsjens think about this..